test/test.js

```diff
@@ -551,6 +551,22 @@ describe('S3rver Tests', function () {
     });
   });
 
+
+  it('should list objects in a bucket filtered by a prefix 2', function (done) {
+    s3Client.listObjectsV2({ 'Bucket': buckets[1], Prefix: 'key' }, function (err, objects) {
+      if (err) {
+        return done(err)
+      }
+      should(objects.Contents.length).equal(4);
+      should.exist(_.find(objects.Contents, {'Key': 'key1'}));
+      should.exist(_.find(objects.Contents, {'Key': 'key2'}));
+      should.exist(_.find(objects.Contents, {'Key': 'key3'}));
+      should.exist(_.find(objects.Contents, {'Key': 'key/key1'}));
```

Comment:
In order to test if the files have been filtered, this test should fail when a file is found not matching the prefix:

```javascript
should.not.exist(_.find(objects.Contents, { 'Key': 'akey1' }))
should.not.exist(_.find(objects.Contents, { 'Key': 'akey2' }))
should.not.exist(_.find(objects.Contents, { 'Key': 'akey3' }))
```

Reply:
The previous test should also test for the same outcome.